packages/meteor-gazelle-wiki/lib/routes.js

```diff
@@ -0,0 +1,60 @@
+WikiController = ApplicationController.extend({
+  waitOn: function () {
+    return Meteor.subscribe('announcements');
+  },
+  data: function () {
+    return {
+      wiki: Articles.find({}, { sort: { createdAt: -1 }})
```

Comment:
Heh, sure. I've already removed this data field and put a more appropriate one into each of the router 'actions' in the `ApplicationController`.

Incidentally, the `ApplicationController` isn't currently being made available through `api.export()` from the `core` package.